db.go

```diff
@@ -148,15 +147,39 @@ func (db *DB) View(fn func(tx *Tx) error) error {
 
 // Backup copies the database to file directory at the given dir.
 func (db *DB) Backup(dir string) error {
-	return db.View(func(tx *Tx) error {
-		return filesystem.CopyDir(db.opt.Dir, dir)
+	return db.View(func(tx *Tx) (err error) {
+		err = tx.db.flock.Unlock()
```

Comment:
If we do not do these lines, backup of nutsdb will be blocked on windows. The reason is the files are locked.